XamlControlsGallery/ControlPages/ButtonPage.xaml

```diff
@@ -19,7 +19,7 @@
         <local:ControlExample x:Name="Example1" HeaderText="A simple Button with text content.">
             <local:ControlExample.Example>
                 <RelativePanel>
-                    <Button x:Name="Button1" Content="Button" Click="Button_Click"
+                    <Button x:Name="Button1" Content="Normal button" Click="Button_Click"
```

Comment:
What about "Standard Xaml Button", "Plain Button", or "Example Button" instead? "Normal" doesn't seem quite right in the context of a UI control. 

CC: @YuliKl

Reply:
Good idea, changed the names a bit so the meaning is clearer.